.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,45 @@
+name: Bug report
+description: Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG: "
+labels: [00 - Bug]
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for taking the time to file a bug report. Before creating a new issue, please make sure to take a few minutes to check the
+      issue tracker for existing issues about the bug.
```

Comment:
Breaking the line here renders this break in the final form:
![Screen Shot 2021-09-17 at 10 16 10 AM](https://user-images.githubusercontent.com/3949932/133788789-949fa86b-fe96-4ba7-9c62-fd69ed4186ae.png)

I believe if you use the character `>` instead of `|` at the `value: ` line, the line break in the source file is not respected when rendering, which should fix this issue. Can you try this out?